### DISCUSSION TITLE
Resend Failed Email Notification

### DISCUSSION THREAD

**Discussion by The-Dan-Main** [solspace/craft-freeform#1483] [Questions & Answers] [answered]
It would be great to have the option to trigger the email notification from the CP, especially if the email could not be sent due to an error.

**Comment by kjmartens** [comment, marked as answer]
Hi @The-Dan-Main,

This is already possible: 🙂

https://docs.solspace.com/craft/freeform/v5/forms/email-notifications/#extra-notifications
